config/api.yml

```diff
@@ -1446,6 +1446,20 @@
         :identifier: instance_edit
       - :name: delete
         :identifier: instance_edit
+  :lans:
+    :description: Lans
+    :options:
+    - :collection
+    :verbs: *g
+    :klass: Lan
+    :collection_actions:
+      :get:
+      - :name: read
+        :identifier: ems_cluster_show_list
```

Comment:
we should probably add the identifiers of other CI's that one can query the lans for, looking at the model,  GET /api/vms and /api/hosts should allow the attributes=lans. Can you verify and update the identifiers accordingly ?  Thanks.